website/docs/d/neptune_engine_version.markdown

```diff
@@ -0,0 +1,40 @@
+---
+subcategory: "Neptune"
+layout: "aws"
+page_title: "AWS: aws_neptune_engine_version"
+description: |-
+  Information about a Neptune engine version.
+---
+
+# Data Source: aws_neptune_engine_version
+
+Information about a Neptune engine version.
+
+## Example Usage
+
+```hcl
+data "aws_neptune_engine_version" "test" {
+  preferred_versions = ["1.0.3.0", "1.0.2.2", "1.0.2.1"]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `engine` - (Optional) DB engine. (Default: `neptune`)
+* `parameter_group_name` - (Optional) The name of a specific DB parameter group family. An example parameter group family is `neptune1`.
+* `preferred_versions` - (Optional) Ordered list of preferred engine versions. The first match in this list will be returned. If no preferred matches are found and the original search returned more than one result, an error is returned.
+* `version` - (Optional) Version of the DB engine. For example, `1.0.1.0`, `1.0.2.2`, and `1.0.3.0`.
```

Comment:
Similar note as #15228 here about mentioning that the default version will be returned otherwise 👍